service/matching/matchingEngine_test.go

```diff
@@ -1602,8 +1603,6 @@ func (s *matchingEngineSuite) TestTaskQueueManagerGetTaskBatch() {
 	s.Nil(err)
 	s.True(0 < len(tasks) && len(tasks) <= rangeSize)
 	s.True(isReadBatchDone)
-
-	tlMgr.engine.removeTaskQueueManager(tlMgr.taskQueueID)
```

Comment:
This is the last line in a test case that tests batch fetching. I couldn't figure out it's purpose and removing it doesn't affect the test outcome. It's either vestigial or just here for coverage.